plugins/CircuitExplorer/CMakeLists.txt

```diff
@@ -124,4 +124,47 @@ set(BRAYNSCIRCUITEXPLORER_LINK_LIBRARIES
 set(BRAYNSCIRCUITEXPLORER_OMIT_LIBRARY_HEADER ON)
 set(BRAYNSCIRCUITEXPLORER_OMIT_VERSION_HEADERS ON)
 set(BRAYNSCIRCUITEXPLORER_OMIT_EXPORT ON)
+
+# Movie generation with FFmpeg
+find_library(AVFORMAT_LIBRARY avformat)
+find_library(AVCODEC_LIBRARY avcodec)
+find_library(AVUTIL_LIBRARY avutil)
+find_library(SWSCALE_LIBRARY swscale)
+
+set(FFMPEG_FOUND ON)
+
+if(NOT AVFORMAT_LIBRARY)
+    message("libavformat not found")
```

Comment:
use `message(STATUS "...")` to keep the aesthetics of the logs

Reply:
I have implemented a new architecture and added the missing docstrings, STATUS and replaced the CMake source selection by a macro, let me know what you think about it.